runtime/compiler/runtime/JITClientSession.hpp

```diff
@@ -27,6 +27,7 @@
 #include "env/PersistentCollections.hpp" // for PersistentUnorderedMap
 #include "il/DataTypes.hpp" // for DataType
 #include "env/VMJ9.h" // for TR_StaticFinalData
+#include "env/JITServerPersistentCHTable.hpp"
```

Comment:
We can forward declare `class JITServerPersistentCHTable` and avoid this include.
We may need an include in the cpp file though

Reply:
Done